diff --git a/node_modules/libp2p/dist/src/connection-manager/index.js b/node_modules/libp2p/dist/src/connection-manager/index.js
index 0ef3ba0..beb74f0 100644
--- a/node_modules/libp2p/dist/src/connection-manager/index.js
+++ b/node_modules/libp2p/dist/src/connection-manager/index.js
@@ -290,7 +290,6 @@ export class DefaultConnectionManager {
         this.outboundPendingConnections++;
         try {
             options.signal?.throwIfAborted();
-            options?.onProgress?.(new CustomProgressEvent('connection:open', peerIdOrMultiaddr));
             const { peerId, multiaddrs } = getPeerAddress(peerIdOrMultiaddr);
             if (this.peerId.equals(peerId)) {
                 throw new InvalidPeerIdError('Can not dial self');
@@ -301,7 +300,6 @@ export class DefaultConnectionManager {
                 if (existingConnection != null) {
                     this.log('had an existing connection to %p as %a', peerId, existingConnection.remoteAddr);
                     options.onProgress?.(new CustomProgressEvent('dial-queue:already-connected'));
-                    options.onProgress?.(new CustomProgressEvent('connection:opened', existingConnection));
                     return existingConnection;
                 }
             }
@@ -335,7 +333,6 @@ export class DefaultConnectionManager {
             if (!trackedConnection) {
                 peerConnections.push(connection);
             }
-            options.onProgress?.(new CustomProgressEvent('connection:opened', connection));
             return connection;
         }
         finally {
